Makefile

```diff
@@ -7,6 +7,12 @@ IMAGE_NAME ?= ceph-csi-operator
 # Allow customization of the name prefix and/or namespace
 NAME_PREFIX ?= ceph-csi-operator-
 NAMESPACE ?= $(NAME_PREFIX)system
+OPERATOR_NAMESPACE = $(shell echo "${$OPERATOR_NAMESPACE}")
```

Comment:
IMO i dont see a need for another variable for operator namespace we can just use `NAMESPACE` in make test

Reply:
@Madhu-1  wrote:
> IMO i dont see a need for another variable for operator namespace we can just use `NAMESPACE` in make test

Agreed, but when I tried to set it directly in the test target definition, I struggled with makefile vs shell syntax because I want to set it conditionally: only when the var is not set in the outside shell env.

Reply:
fixed to set it unconditionally

Reply:
if someone wants to set the NAMESPACE for the operator to run the test they can set the `NAMESPACE` that we already have, we should not two different variables to serve the same purpose to set the NAMESPACE for the operator.